cvat-ui/src/reducers/settings-reducer.ts

```diff
@@ -28,6 +28,7 @@ const defaultState: SettingsState = {
         autoSave: false,
         autoSaveInterval: 15 * 60 * 1000,
         aamZoomMargin: 100,
+        showObjectsTextAlways: true,
```

Comment:
Why is it a good idea to enable this option by default?

Reply:
I did it for test. Forgot to disable.